### ISSUE TITLE
How to exclude PDFs from being downloaded

### ISSUE THREAD

**Issue by TheOttoman2012** [mylar3/mylar3#1245]
I want to exclude PDFs from being downloaded when Mylar searches. How would I go about doing this?

**Comment by evilhero** [comment]
You can't - Mylar will only download cbr or cbz files but it's based on what the given filename the indexer returns. 

If it's downloading files that aren't that, then the indexer it's downloading from is incorrectly labelling the file or leaving out the extension (typical with torrents). 

When Mylar goes to download it, it doesn't know what the extension is until after it downloads. So while it downloads successfully, when it goes to post-process the file it won't be able to as the extension is an invalid type and gets rejected at that point. 

Basically, the indexervits downloading from is crap for comics (su has been bad for this if I recall, as well as torrents generally). You might want to consider new ones that focus more on comics as that will decrease these types of problems from happening.

**Comment by evilhero** [comment]
update to the latest version on either branch - go into the Configuration settings and the 2nd last tab (Quality & Post-Processing), and on the left side you'll see an ``Ignore Search Words Restrictions``. 

``pdf`` should be already present by default, but if it's not there, just add it and save the configuration for the settings to take effect.

**Comment by evilhero** [comment]
Closed as completed.
